examples/event_handler_rest/src/micro_function_all_users_route.py

```diff
@@ -0,0 +1,65 @@
+import json
+from http import HTTPStatus
+
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools.event_handler import APIGatewayRestResolver, Response
+from aws_lambda_powertools.utilities.parser import BaseModel
```

Comment:
```suggestion
from dataclasses import dataclass
```

We should remove Pydantic to make this example easier to test and not force customers to bring external dependencies. We can use dataclass instead. What do you think?